src/Common/Core/Form.php

```diff
@@ -236,6 +236,39 @@ public function addTime($name, $value = null, $class = null, $classError = null)
         // create and return a time field
         return parent::addTime($name, $value, $class, $classError);
     }
+
+    /**
+     * @return string|null
+     */
+    public static function getUploadMaxFileSize()
+    {
+        $uploadMaxFileSize = ini_get('upload_max_filesize');
+        if ($uploadMaxFileSize === false) {
+            return null;
+        }
+
+        // reformat if defined as an integer
+        if (\SpoonFilter::isInteger($uploadMaxFileSize)) {
```

Comment:
As we try to eliminate Spoon, wouldn't it be more logical to use: http://php.net/manual/en/function.is-numeric.php instead of SpoonFilter::isInteger?